Feature/narre metplus5

BinbinZhou-NOAA opened the pull request:
## Pull Request Testing ##

- [y ] Describe testing already performed for this Pull Request:</br>

- [y ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

- [ y] Has the code been checked to ensure that no errors occur during the execution? **[Yes or No]**

- [ y] Do these updates/additions include sufficient testing updates? **[Yes or No]**

- [y ] Please complete this pull request review by **[Fill in date]**.</br>

## Pull Request Checklist ##

- [ y] Review the source issue metadata (required labels, projects, and milestone).
- [ y] Complete the PR description above.
- [y] Ensure the PR title matches the feature branch name.
- [y ] Check the following:
- [y ]  Instructions provided on how to run
- [y ]  Developer's name is replaced by ${user} where necessary throughout the code
- [y ]  Check that the ecf file has all the proper definitions of variables
- [y ]  Check that the jobs file has all the proper settings of COMIN and COMOUT and other input variables
- [y ]  Check to see that the output directory structure is followed
- [ y]  Be sure that you are not using MET utilities outside the METplus wrapper structure

- [y ] After submitting the PR, select **Development** issue with the original issue number.
- [y ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ y] Close the linked issue.

PerryShafran-NOAA committed:
Update README.md

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/narre_metplus5

BinbinZhou-NOAA committed:
First update

BinbinZhou-NOAA committed:
update run.ver

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/narre_metplus5

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/narre_metplus5

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/narre_metplus5

PerryShafran-NOAA commented:
Starting work on this PR.  

Perry

PerryShafran-NOAA commented on ecf/narre/plots/evs_narre_plot.o52404969:
The .o files should not be part of the PR and should not be posted to any github repository.

BinbinZhou-NOAA replied:
Just removed it from the fork 

Thanks!

PerryShafran-NOAA commented on ecf/narre/plots/submit_all_plots.output:
Remove this output file from the repository.

PerryShafran-NOAA commented on ecf/narre/plots/submit_all_plots.sh:
Remove this file from the repository.

PerryShafran-NOAA commented on ecf/narre/stats/instruction_test.txt:
Remove this file from the repository.  Instructions should be located within the PR.

BinbinZhou-NOAA replied:
OK

PerryShafran-NOAA commented on ecf/narre/stats/jevs_narre_stats.ecf:
We should not be using /bin/bash -l in the top line of the scripts.  This is now required by NCO to reduce the load on the system.

BinbinZhou-NOAA replied:
yes

PerryShafran-NOAA replied:
Thanks! Could you make this change and recommit?

PerryShafran-NOAA commented on ecf/narre/stats/run_narre_job.output:
Remove this output file from the repository.

BinbinZhou-NOAA replied:
yes, remove it

PerryShafran-NOAA replied:
Please make the change and recommit to the repository.

PerryShafran-NOAA commented on ecf/narre/stats/submit_narre_job.sh:
Remove file from repository.  This is not needed.

PerryShafran-NOAA commented on parm/metplus_config/machine.conf:
Check to see that this is a modification from the authoritative machine.conf.  The authoritative machine.conf should be used.

BinbinZhou-NOAA replied:
OK, please go ahead to use the version in the auth repo

PerryShafran-NOAA replied:
Please be sure you have the auth repo version in your feature branch, then recommit.  

Thanks!

PerryShafran-NOAA commented on scripts/narre/plots/exevs_narre_plots.sh:
Why are we moving from sh to ksh?

BinbinZhou-NOAA replied:
either sh or ksh or bash are OK

PerryShafran-NOAA commented on ush/narre/ush_narre_plot_py/__pycache__/check_variables.cpython-38.pyc:
The pycache directory should not be included in the github repository.  Binaries are supposed to be on github.

PerryShafran-NOAA replied:
The pycache directory is removed.

PerryShafran-NOAA commented on versions/run.ver:
Check to see that you are using the authoritative run.ver.  If there are items that need to be added, that are not there, please indicate if that is the case.

BinbinZhou-NOAA replied:
No adding is needded, so use auth's run.ver

PerryShafran-NOAA replied:
Please bring in the auth run.ver to be sure that we are using it.

BinbinZhou-NOAA committed:
delete *o file

PerryShafran-NOAA commented on ecf/narre/plots/jevs_narre_past31days_plots.ecf:
I notice that your have the following line in here:

`export ush_dir=/lfs/h2/emc/vpppg/noscrub/marcel.caron/verif_plotting/ush`

We cannot refer to codes or files outside the EVS system.  We cannot also have individual names in any EVS script in this manner.  All of the ush files for plotting should be in the ush/narre directory.  

I do notice that there are also many of the plotting py files in your own ush/narre directory.  Please clarify that your EVS plotting script uses these ush files and not the ones that point to Marcel's directory.

BinbinZhou-NOAA replied:
This line is just for testing can be removed since usr_dir is overwritten in the job file

PerryShafran-NOAA replied:
OK, thanks!  Please remove it and resubmit this file.

ShelleyMelchior-NOAA commented on ecf/narre/stats/run_narre_job.output:
How about the *.output files? I think these also should not be in the code repository.

ShelleyMelchior-NOAA commented on versions/run.ver:
```diff
@@ -1,3 +1,6 @@
+
+ 
+
```
Can we remove these 3 blank lines that were introduced?

BinbinZhou-NOAA replied:
yes

PerryShafran-NOAA replied:
Please make the changes and recommit.  

Thanks!

ShelleyMelchior-NOAA commented:
Let's also clean out the binary directory in ush: `ush/narre/ush_narre_plot_py/__pycache__`

BinbinZhou-NOAA commented:
Yes, *.output file should be removed as well


On Fri, Apr 7, 2023 at 11:23 AM Shelley Melchior ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> On ecf/narre/stats/run_narre_job.output
> <https://github.com/NOAA-EMC/EVS/pull/94#discussion_r1160776327>:
>
> How about the *.output files? I think these also should not be in the code
> repository.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/94#pullrequestreview-1376311931>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFLTB3UW4RPVPD5XX43XAAWP5ANCNFSM6AAAAAAWVW4IXM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

BinbinZhou-NOAA committed:
delete *output filse

BinbinZhou-NOAA commented:
Not sure  __pycache__ directory is required or not. I just copied it from
Marcel's plotting directory.
Let me ask Marcel about it.

Binbin

On Fri, Apr 7, 2023 at 11:25 AM Shelley Melchior ***@***.***>
wrote:

> Let's also clean out the binary directory in ush:
> ush/narre/ush_narre_plot_py/__pycache__
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/94#issuecomment-1500387659>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFJK5JHTTCL4L3PIFQLXAAWXRANCNFSM6AAAAAAWVW4IXM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

malloryprow commented:
@BinbinZhou-NOAA It isn't need. Python will create it, if needed, during the execution of python scripts.

https://stackoverflow.com/questions/16869024/what-is-pycache

BinbinZhou-NOAA commented:
OK, then it should be deleted.

Thanks!

Binbin





On Fri, Apr 7, 2023 at 11:39 AM Mallory Row ***@***.***>
wrote:

> @BinbinZhou-NOAA <https://github.com/BinbinZhou-NOAA> It isn't need.
> Python will create it, if needed, during the execution of python scripts.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/94#issuecomment-1500397868>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFOS77GOJGDKLSAG4SLXAAYJPANCNFSM6AAAAAAWVW4IXM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

PerryShafran-NOAA commented:
pycache is created for the execution of the python scripts, but because they are binary files, they can't be included in the github repository, much like exec files cannot be.

BinbinZhou-NOAA committed:
delete __pycache__ sub-directory

BinbinZhou-NOAA commented:
OK, it has been removed in the folk

Binbin

On Fri, Apr 7, 2023 at 11:43 AM PerryShafran-NOAA ***@***.***>
wrote:

> pycache is created for the execution of the python scripts, but because
> they are binary files, they can't be included in the github repository,
> much like exec files cannot be.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/94#issuecomment-1500400703>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFMS4Q5ROHLJ3LVGTFLXAAYYVANCNFSM6AAAAAAWVW4IXM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

BinbinZhou-NOAA committed:
new update

BinbinZhou-NOAA commented:
I have fixed all of the above  issues and updated relevant files in the
github fork.

Binbin

On Fri, Apr 7, 2023 at 3:05 PM PerryShafran-NOAA ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> On ecf/narre/plots/jevs_narre_past31days_plots.ecf
> <https://github.com/NOAA-EMC/EVS/pull/94#discussion_r1160902548>:
>
> OK, thanks! Please remove it and resubmit this file.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/94#discussion_r1160902548>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFNNPYUS3TUPUBHMRO3XABQPZANCNFSM6AAAAAAWVW4IXM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

BinbinZhou-NOAA commented:
Part 1/2:  Instruction for stats part testing:

   cd  /lfs/h2/emc/vpppg/noscrub/$USER/EVS/evs.v1.0/ecf/narre/stats

type following
   qsub jevs_narre_stats.ecf

   Note: since "#PBS -j oe" is set, the stdout file will be in the submit directory
     /lfs/h2/emc/vpppg/noscrub/$USER/EVS/evs.v1.0/ecf/narre/stats

Note, if want to run interactive mode, edit jevs_narre_stats.ecf file by changing
export run_mpi=yes --> export run_mpi=no, then type
jevs_narre_stats.ecf

The running directory (i.e. $DATA) will be in /lfs/h2/emc/ptmp/${USER}/evs/tmpnwprd
evs_narre_grid2obs_stats.$jobid

jobid is automatically created ($$) during the run

The running has 3 stages
 (1) Run METplus to process PB2NC prepbufr files
       After finished, the processed prepbufr data are in sub-directory $DATA/prepbufr.20221002

 (2) Run METplus to generate stat files
     This step is to verify  NARRE's ensemble mean by PointStat tool

       All of output for stat genration is in the sub-directory grid2obs

       After the stage (1) is finished, following 8 job scripts are generated for MPI run
        run_narre_narremean.CONUS.range1.sh
        run_narre_narremean.CONUS.range2.sh
        run_narre_narremean.CONUS.range3.sh
        run_narre_narremean.CONUS.range4.sh
        run_narre_narremean.Alaska.range1.sh
        run_narre_narremean.Alaska.range2.sh
        run_narre_narremean.Alaska.range3.sh
        run_narre_narremean.Alaska.range4.sh


       The METplus output_base for each jobs are in grid2obs
         such as run_narre_narremean.CONUS.range1

       In each job's output_base directory, there are 3 sub directories
          logs, stats, and tmp

        The small stat files are stored in the stats.

 (3) Combine the small stat files to a large stat file

     After the small stat files of all of the 8 jobs are created, they will be copied to
    the directory
       /lfs/h2/emc/vpppg/noscrub/${USER}/com/evs/v1.0/stats/narre/atmos.$VDATE

   Thes small stat files then are combined (by StatAnalysis tool) into a large stat file
   called NARRE_MEAN_grid2obs_20221002.stat
   This large stat file is stored in the directory
      /lfs/h2/emc/vpppg/noscrub/binbin.zhou/com/evs/v1.0/stats/narre.$VDATE

   The StatAnalysis METplus output_base directory is $DATA/gather, in which all of running info are stored

Part 2/2:  Instruction for plot par testing

Testing ecf directory:
  /lfs/h2/emc/vpppg/noscrub/$USER/EVS/ecf/narre/plots

  There are 4 ecf files:
  evs_narre_past31days_plots.ecf
  evs_narre_past90days_plots.ecf
   These 2 ecf jobs are independent and can be submit at same time


Submit jobs
 qsub  evs_narre_past31days_plots.ecf
 qsub  evs_narre_past90days_plots.ecf

  Note: since "#PBS -j oe" is set, the stdout file will be in the submit directory
   /lfs/h2/emc/vpppg/noscrub/$USER/EVS/ecf/narre/plots

The running directory (i.e. $DATA) will be in /lfs/h2/emc/ptmp/${USER}/evs/tmpnwprd

There is a script called submit_all_plots.sh. By running this script, all 4 jobs
can be submitted.

The flowchart for each job is similar:

ecf scripts in /lfs/h2/emc/vpppg/noscrub/binbin.zhou/EVS/ecf/narre

The generated plots are tarred in one tar files for each job and is sent to the COMOUT directory:
   /lfs/h2/emc/vpppg/noscrub/${USER}/com/evs/v1.0/plots/narre/atmos.$VDATE
    plots.narre.grid2obs.last31days.v${VDATE}.tar
    plots.narre.grid2obs.last90days.v${VDATE}.tar

PerryShafran-NOAA commented:
Hi, Binbin,

Please be sure that all your plots tar files start with "evs":  

Your tar filename should be:  evs.plots.narre.grid2obs.last31days.v${VDATE}.tar.  

Can you be sure that you change the filenames so they have the proper naming structure?

Perry

PerryShafran-NOAA commented:
Also, looking into your NARRE tar file, you have only 4 plots:

```
30 (dlogin04) /lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/plots/narre/atmos.20230409 > tar -tvf plots.narre.grid2obs.last31days.v20230409.tar
-rw-r--r-- perry.shafran/emc 397879 2023-04-10 13:19 evs.narre.ctc.ceiling_l0.last31days.perfdiag.f012.alaska.png
-rw-r--r-- perry.shafran/emc 399754 2023-04-10 13:19 evs.narre.ctc.ceiling_l0.last31days.perfdiag.f012.buk_conus.png
-rw-r--r-- perry.shafran/emc 390822 2023-04-10 13:24 evs.narre.ctc.vis_l0.last31days.perfdiag.f012.alaska.png
-rw-r--r-- perry.shafran/emc 397169 2023-04-10 13:26 evs.narre.ctc.vis_l0.last31days.perfdiag.f012.buk_conus.png
31 (dlogin04) /lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/plots/narre/atmos.20230409
```

Is this correct?

Thanks!

Perry

BinbinZhou-NOAA commented:
Only individual graphic file naming is defined but the combined  graphic
tar file naming is not defined in this document
https://docs.google.com/document/d/1ZVfHzhzLnyDpqVoE0hB4mAluUDZ87W5hIqsw03lTLXY/edit

No problem,  I'll update them. Please wait for this update in the narre
github folk

Binbin

Thanks!

Binbin

On Mon, Apr 10, 2023 at 9:41 AM PerryShafran-NOAA ***@***.***>
wrote:

> Hi, Binbin,
>
> Please be sure that all your plots tar files start with "evs":
>
> Your tar filename should be:
> evs.plots.narre.grid2obs.last31days.v${VDATE}.tar.
>
> Can you be sure that you change the filenames so they have the proper
> naming structure?
>
> Perry
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/94#issuecomment-1501825607>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFPK745GD6DHPS5S6EDXAQEWNANCNFSM6AAAAAAWVW4IXM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

PerryShafran-NOAA commented:
See this doc for the tar naming instructions:

https://docs.google.com/document/d/1HOJ5hTCMz-RipkGLk8zd2dov3V5IbhtgGJocqbwGQX8/edit

Perry

BinbinZhou-NOAA commented:
Yes, these are correct, only 4 plots for narre, 2 for ceiling performance
diagram and 2 for visibility diagram.
See graphics sheet
https://docs.google.com/spreadsheets/d/1rumS6wwSD1DDFKF8of9z57YQ5dh4rlX44f25p6d02Wg/edit?pli=1#gid=1503813652

Binbin

On Mon, Apr 10, 2023 at 9:44 AM PerryShafran-NOAA ***@***.***>
wrote:

> Also, looking into your NARRE tar file, you have only 4 plots:
>
> 30 (dlogin04) /lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/plots/narre/atmos.20230409 > tar -tvf plots.narre.grid2obs.last31days.v20230409.tar
> -rw-r--r-- perry.shafran/emc 397879 2023-04-10 13:19 evs.narre.ctc.ceiling_l0.last31days.perfdiag.f012.alaska.png
> -rw-r--r-- perry.shafran/emc 399754 2023-04-10 13:19 evs.narre.ctc.ceiling_l0.last31days.perfdiag.f012.buk_conus.png
> -rw-r--r-- perry.shafran/emc 390822 2023-04-10 13:24 evs.narre.ctc.vis_l0.last31days.perfdiag.f012.alaska.png
> -rw-r--r-- perry.shafran/emc 397169 2023-04-10 13:26 evs.narre.ctc.vis_l0.last31days.perfdiag.f012.buk_conus.png
> 31 (dlogin04) /lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/plots/narre/atmos.20230409
>
> Is this correct?
>
> Thanks!
>
> Perry
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/94#issuecomment-1501828479>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFIJAGH6EBVKMX47HXDXAQFDBANCNFSM6AAAAAAWVW4IXM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

PerryShafran-NOAA commented:
OK, thanks, good to know!

Perry

BinbinZhou-NOAA commented:
Sorry should be 8 plots, 4 for 31days and 4 for 90 days

Binbin

On Mon, Apr 10, 2023 at 10:11 AM Binbin Zhou - NOAA Affiliate <
***@***.***> wrote:

> Yes, these are correct, only 4 plots for narre, 2 for ceiling performance
> diagram and 2 for visibility diagram.
> See graphics sheet
>
> https://docs.google.com/spreadsheets/d/1rumS6wwSD1DDFKF8of9z57YQ5dh4rlX44f25p6d02Wg/edit?pli=1#gid=1503813652
>
> Binbin
>
> On Mon, Apr 10, 2023 at 9:44 AM PerryShafran-NOAA <
> ***@***.***> wrote:
>
>> Also, looking into your NARRE tar file, you have only 4 plots:
>>
>> 30 (dlogin04) /lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/plots/narre/atmos.20230409 > tar -tvf plots.narre.grid2obs.last31days.v20230409.tar
>> -rw-r--r-- perry.shafran/emc 397879 2023-04-10 13:19 evs.narre.ctc.ceiling_l0.last31days.perfdiag.f012.alaska.png
>> -rw-r--r-- perry.shafran/emc 399754 2023-04-10 13:19 evs.narre.ctc.ceiling_l0.last31days.perfdiag.f012.buk_conus.png
>> -rw-r--r-- perry.shafran/emc 390822 2023-04-10 13:24 evs.narre.ctc.vis_l0.last31days.perfdiag.f012.alaska.png
>> -rw-r--r-- perry.shafran/emc 397169 2023-04-10 13:26 evs.narre.ctc.vis_l0.last31days.perfdiag.f012.buk_conus.png
>> 31 (dlogin04) /lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/plots/narre/atmos.20230409
>>
>> Is this correct?
>>
>> Thanks!
>>
>> Perry
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/NOAA-EMC/EVS/pull/94#issuecomment-1501828479>, or
>> unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AQBMPFIJAGH6EBVKMX47HXDXAQFDBANCNFSM6AAAAAAWVW4IXM>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>
>
> --
>
> Binbin Zhou
>
> Physical Scientist
>
> Lynker at NOAA/NWS/NCEP/EMC
>
> 5830 University Research Ct.
>
> College Park, MD 20740
>
> ***@***.***
>
> 301-683-3683
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

PerryShafran-NOAA commented:
Right - 4 for each tar file.

Perry

BinbinZhou-NOAA committed:
update plot script

BinbinZhou-NOAA commented:
Perry,
  I found a bug in the plotting 31days ecf file:
#PBS - j oe
This should be #PBS -j oe
Don't why got this extra space. This is why no 31days tar file is generated
in your testing.
I have corrected it
The tar file names are  also changed as you suggested.
Both above have been updated in github fork feature/narre_metplus5.

Please update them in your test code.

Binbin


On Mon, Apr 10, 2023 at 10:13 AM PerryShafran-NOAA ***@***.***>
wrote:

> OK, thanks, good to know!
>
> Perry
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/94#issuecomment-1501867541>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFJJ4YQL56ERKVH3YNLXAQIQ5ANCNFSM6AAAAAAWVW4IXM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

PerryShafran-NOAA commented:
Thanks!  I will do one last test and then have you validate.

Perry

PerryShafran-NOAA commented:
Regarding the plots, I looked at the plot spreadsheet and the temporal attribute says "0-12h/1h", which would seem to indicate that you should have 12 plots for each variable (one for the 0hr fcst, one for the 1hr fcst, and so on to the 12hr fcst), 24 plots all told, not just 1 plot for the 12hr forecast.  @AliciaBentley-NOAA, @LoganDawson-NOAA  - do I have this wrong?  I just wanted to make sure before I approve.

Perry

AliciaBentley-NOAA commented:
Hi Perry. These plots are Performance Diagrams, so either all 12 hours
should be plotted on each Performance Diagram as 12 individual points or
each hour could be shown on its own Performance Diagram. I would be OK with
all 12 points on each Performance Diagram, but I have not seen the NARRE
plots before (North American model). What do they currently look like to
you?

___________________________________
*Alicia M. Bentley, Ph.D.*
Physical Scientist | Model Evaluation Group
<https://www.emc.ncep.noaa.gov/users/meg/home/>
Global Model Verification Team Lead
NOAA/NWS/NCEP/EMC (VPPPG Branch)
Work Phone: ‪(301) 683-3783‬


On Mon, Apr 10, 2023 at 10:59 AM PerryShafran-NOAA ***@***.***>
wrote:

> Regarding the plots, I looked at the plot spreadsheet and the temporal
> attribute says "0-12h/1h", which would seem to indicate that you should
> have 12 plots for each variable (one for the 0hr fcst, one for the 1hr
> fcst, and so on to the 12hr fcst), 24 plots all told, not just 1 plot for
> the 12hr forecast. @AliciaBentley-NOAA
> <https://github.com/AliciaBentley-NOAA>, @LoganDawson-NOAA
> <https://github.com/LoganDawson-NOAA> - do I have this wrong? I just
> wanted to make sure before I approve.
>
> Perry
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/94#issuecomment-1501920530>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APUWOBG6GRZQ7OO4QXXVW33XAQN45ANCNFSM6AAAAAAWVW4IXM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

LoganDawson-NOAA commented:
@PerryShafran-NOAA the way Binbin has set up these four plots is how I envisioned them. Performance diagrams are a good way to compile a lot of statistical information on the same plot, so I don't think it's overly useful to have a separate performance diagram for each forecast hour from F00 to F12. The key for the temporal attribute is that we want to include stats for F00-F12 from all forecast cycles.

@BinbinZhou-NOAA the only issue I see with your plots is that third line of the plot title is very long and doesn't format well. Is there any easy way to see up that line to say "Valid every 3 hours for 09 Mar 2023 to 09 Apr 2023, Forecast Hours 1-12"?

PerryShafran-NOAA commented:
OK, Logan, Alicia, that makes sense.  Then we have the correct number of plots.  Thanks!

Perry

LoganDawson-NOAA commented:
@AliciaBentley-NOAA for performance diagrams in general, the key information we want to see is the statistical performance at different forecast thresholds. If we want to look at metrics based on forecast lead, then we would need to generate sets of fhr_mean plots at each different threshold

AliciaBentley-NOAA commented:
Thank you, Logan! The graphics spreadsheet doesn't list the NARRE
thresholds, and I admit that I didn't go back and look at the stats
spreadsheet to see them. Without the thresholds listed, I couldn't really
picture these Performance Diagrams correctly, but what you all have done
here makes sense now. Thanks!

___________________________________
*Alicia M. Bentley, Ph.D.*
Physical Scientist | Model Evaluation Group
<https://www.emc.ncep.noaa.gov/users/meg/home/>
Global Model Verification Team Lead
NOAA/NWS/NCEP/EMC (VPPPG Branch)
Work Phone: ‪(301) 683-3783‬


On Mon, Apr 10, 2023 at 11:27 AM Logan Dawson ***@***.***>
wrote:

> @AliciaBentley-NOAA <https://github.com/AliciaBentley-NOAA> for
> performance diagrams in general, the key information we want to see is the
> statistical performance at different forecast thresholds. If we want to
> look at metrics based on forecast lead, then we would need to generate sets
> of fhr_mean plots at each different threshold
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/94#issuecomment-1501955774>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APUWOBCQK2GADVQIGTXUKLTXAQRHTANCNFSM6AAAAAAWVW4IXM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

LoganDawson-NOAA commented:
@BinbinZhou-NOAA I just noticed a few issues with the titles and labels on these NARRE plots. Look for a separate email with comments from me.

BinbinZhou-NOAA committed:
Update ecf and scripts

BinbinZhou-NOAA committed:
update machine.conf

BinbinZhou-NOAA committed:
remove *.sh in ecf

BinbinZhou-NOAA committed:
change G214 to G242 for Alaska

BinbinZhou-NOAA commented:
Perry,

  I just updated a few files  in the github fork feature/narre_metplus5:
(1) Removed all  *sh files in the ecf directories
(2) Updated machine.conf to be same as that in the autho's repo (just added
one line to the old file)
(3) Changed the Alaska grid from G214 to G242
     This issue was found by Logan. He is correct. The grid# should br
G242 instead of G214
Please update them in your testing

Thanks!

Binbin



On Mon, Apr 10, 2023 at 11:39 AM Logan Dawson ***@***.***>
wrote:

> @BinbinZhou-NOAA <https://github.com/BinbinZhou-NOAA> I just noticed a
> few issues with the titles and labels on these NARRE plots. Look for a
> separate email with comments from me.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/94#issuecomment-1501967302>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFJ2DHCU3GBELX5HHT3XAQSR3ANCNFSM6AAAAAAWVW4IXM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

BinbinZhou-NOAA committed:
change COMIN, COMOUT

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/narre_metplus5

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/narre_metplus5

BinbinZhou-NOAA committed:
add missing data alert

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/narre_metplus5

BinbinZhou-NOAA committed:
Change run.ver to run.ver.metplus5.0.0

BinbinZhou-NOAA committed:
update RAP prepbufr file name

BinbinZhou-NOAA committed:
Update to reset jobid and remove sh in execution of scripts

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/narre_metplus5

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/narre_metplus5

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/narre_metplus5

BinbinZhou-NOAA committed:
change Pb2nc_obsRAP_Prepbufr.cong to Pb2nc_obsRAP_Prepbufr.conf

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/narre_metplus5

BinbinZhou-NOAA committed:
update py scripts by adding top head

PerryShafran-NOAA commented:
Testing is now underway for the stats job.

BinbinZhou-NOAA commented:
Perry,

  I am glad to hear this. If the stat and plot files are output, I can
check them.
Tomorrow and Friday I will take 2 days off and may not check them

Binbin

On Wed, May 10, 2023 at 3:54 PM PerryShafran-NOAA ***@***.***>
wrote:

> Testing is now underway for the stats job.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/94#issuecomment-1542725402>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFP74KLQOZVQMJ7J22LXFPW53ANCNFSM6AAAAAAWVW4IXM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

PerryShafran-NOAA commented:
I had to re-submit because the job went past the wall clock time of 30 minutes.  Could be because I hear that Cactus is running slow again.  I'm afraid that we won't be able to examine results today.  Hopefully I can get the jobs in tomorrow, and you can validate when you return from your leave.

Perry

BinbinZhou-NOAA commented:
Perry,

   I also got same issue for my submitted jobs, waiting for a very long
time.
Sure I'll check them next week if there are nothing output today.

Binbin


On Wed, May 10, 2023 at 4:46 PM PerryShafran-NOAA ***@***.***>
wrote:

> I had to re-submit because the job went past the wall clock time of 30
> minutes. Could be because I hear that Cactus is running slow again. I'm
> afraid that we won't be able to examine results today. Hopefully I can get
> the jobs in tomorrow, and you can validate when you return from your leave.
>
> Perry
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/94#issuecomment-1542787131>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFJVEANHKHE26AWECW3XFP5AXANCNFSM6AAAAAAWVW4IXM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

AliciaBentley-NOAA commented:
Hi all! Cactus was very slow last week, preventing us from getting reasonable job run times. Did these NARRE jobs run faster for you today?

BinbinZhou-NOAA commented:
My narre run was set to run at an early time,  so it was completed in time.

Binbin

On Mon, May 15, 2023 at 2:43 PM Alicia Bentley ***@***.***>
wrote:

> Hi all! Cactus was very slow last week, preventing us from getting
> reasonable job run times. Did these NARRE jobs run faster for you today?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/94#issuecomment-1548371330>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFMOZCBJNOAQUB4WNK3XGJ2MBANCNFSM6AAAAAAWVW4IXM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

PerryShafran-NOAA commented:
Running another test of the NARRE today.  

Perry

BinbinZhou-NOAA commented:
Perry,

  I have checked your stat and plot files, all are fine and same as mine.

Thanks!

Binbin

On Wed, May 17, 2023 at 10:26 AM PerryShafran-NOAA ***@***.***>
wrote:

> Running another test of the NARRE today.
>
> Perry
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/94#issuecomment-1551502120>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFLTNTPG5E5DXERSPHLXGTNZBANCNFSM6AAAAAAWVW4IXM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

PerryShafran-NOAA commented on ecf/narre/plots/jevs_narre_past31days_plots.ecf:
```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+#PBS -N jevs_narre_plot
+#PBS -j oe
```
Hi, Binbin,

One last item.  Would you change the PBS -N option in your plot job card to jevs_narre_past31days_plots and jevs_narre_past90days_plots?  Just as before, these correspond to the names of the output files, so they need to be different so we (and NCO) can differentiate them.

Thanks!

Perry

PerryShafran-NOAA replied:
@BinbinZhou-NOAA This is the one change that I needed you to address.

Perry

PerryShafran-NOAA replied:
@AliciaBentley-NOAA here.

This is a minor issue that doesn't require a re-run.  Shelley and I believe that we can merge the code now and address this later.

PerryShafran-NOAA replied:
I will approve this and we can deal with this later.

Perry

AliciaBentley-NOAA commented:
Hi everyone,

With 10 business days left until the start of the EVSv1 Beta Test (with two
MET-11.0.0 ensemble PRs and four EVS VE ensemble PRs left to do), I'm going
to push this to the top of your inboxes again. ☺

Shelley/Perry- Is this PR ready to be merged? If so, can the global_ens PR
be started this week? The global_ens PR might take longer than the other
ensemble PRs because of its size and complexity. Thanks!

Binbin- Please start preparing PRs that transition HREF and SREF to
MET-11.0.2/METplus-5.0.1 and the EVS VE and submit them whenever you are
able to! And thank you again for your recent work on NAEFSv7. We appreciate
it.

Take care,
Alicia
___________________________________
*Alicia M. Bentley, Ph.D.*
Global Verification Project Lead
Model Evaluation Group (MEG) <https://www.emc.ncep.noaa.gov/users/meg/home/>
NOAA/NWS/NCEP/EMC (VPPPG Branch)
Work Phone: ‪(301) 683-3783‬


On Wed, May 17, 2023 at 12:10 PM BinbinZhou-NOAA ***@***.***>
wrote:

> Perry,
>
> I have checked your stat and plot files, all are fine and same as mine.
>
> Thanks!
>
> Binbin
>
> On Wed, May 17, 2023 at 10:26 AM PerryShafran-NOAA ***@***.***>
> wrote:
>
> > Running another test of the NARRE today.
> >
> > Perry
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/NOAA-EMC/EVS/pull/94#issuecomment-1551502120>, or
> > unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AQBMPFLTNTPG5E5DXERSPHLXGTNZBANCNFSM6AAAAAAWVW4IXM
> >
> > .
> > You are receiving this because you were mentioned.Message ID:
> > ***@***.***>
> >
>
>
> --
>
> Binbin Zhou
>
> Physical Scientist
>
> Lynker at NOAA/NWS/NCEP/EMC
>
> 5830 University Research Ct.
>
> College Park, MD 20740
>
> ***@***.***
>
> 301-683-3683
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/94#issuecomment-1551690626>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APUWOBDYXDRAFW3TETXKZGTXGT2AXANCNFSM6AAAAAAWVW4IXM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

PerryShafran-NOAA commented:
Hi, Alicia,

The stats and plots have been validated, but there were a few minor corrections that I wanted Binbin to make in the PBS function.  If that's not important enough to worry about, I can merge them now.  I plan to start on the global_ens PR today.  Please let me know if you are OK with merging now and allowing this issue to be resolved later (it can be resolved when I put in the parallel).  

Thanks!

Perry

AliciaBentley-NOAA commented:
Hi Perry,

If Binbin needs to make changes still, then he should make those changes.
Are the changes you are waiting for included in this thread? I think I see
some requested changes from over a month ago. Would this PR need to be
re-tested after those changes are made?

Binbin- can you please make the changes that Perry requested of you in
early April? Thanks!

Take care,
Alicia
___________________________________
*Alicia M. Bentley, Ph.D.*
Global Verification Project Lead
Model Evaluation Group (MEG) <https://www.emc.ncep.noaa.gov/users/meg/home/>
NOAA/NWS/NCEP/EMC (VPPPG Branch)
Work Phone: ‪(301) 683-3783‬


On Thu, May 18, 2023 at 9:46 AM PerryShafran-NOAA ***@***.***>
wrote:

> Hi, Alicia,
>
> The stats and plots have been validated, but there were a few minor
> corrections that I wanted Binbin to make in the PBS function. If that's not
> important enough to worry about, I can merge them now. I plan to start on
> the global_ens PR today. Please let me know if you are OK with merging now
> and allowing this issue to be resolved later (it can be resolved when I put
> in the parallel).
>
> Thanks!
>
> Perry
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/94#issuecomment-1553081939>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APUWOBC7ZC3WYRVJ46RDZ4LXGYR3BANCNFSM6AAAAAAWVW4IXM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

ShelleyMelchior-NOAA commented:
I am ok w/ the PBS item going unaddressed for this PR. Since @BinbinZhou-NOAA needs to make follow on PRs for version updates, he can make the PBS changes in those.

PerryShafran-NOAA commented:
Hi, Alicia,

The early April ones have been addressed.  The only outstanding issue is one that I posted just yesterday.  It's a minor one that doesn't need a rerun.  

Thanks!

Perry

BinbinZhou-NOAA commented:
Alicia, Perry,

   Sorry I can not catch up what changes need to be made? Please show me
again.

Thanks!

Binbin


On Thu, May 18, 2023 at 10:05 AM Alicia Bentley ***@***.***>
wrote:

> Hi Perry,
>
> If Binbin needs to make changes still, then he should make those changes.
> Are the changes you are waiting for included in this thread? I think I see
> some requested changes from over a month ago. Would this PR need to be
> re-tested after those changes are made?
>
> Binbin- can you please make the changes that Perry requested of you in
> early April? Thanks!
>
> Take care,
> Alicia
> ___________________________________
> *Alicia M. Bentley, Ph.D.*
> Global Verification Project Lead
> Model Evaluation Group (MEG) <
> https://www.emc.ncep.noaa.gov/users/meg/home/>
> NOAA/NWS/NCEP/EMC (VPPPG Branch)
> Work Phone: ‪(301) 683-3783‬
>
>
> On Thu, May 18, 2023 at 9:46 AM PerryShafran-NOAA ***@***.***>
> wrote:
>
> > Hi, Alicia,
> >
> > The stats and plots have been validated, but there were a few minor
> > corrections that I wanted Binbin to make in the PBS function. If that's
> not
> > important enough to worry about, I can merge them now. I plan to start on
> > the global_ens PR today. Please let me know if you are OK with merging
> now
> > and allowing this issue to be resolved later (it can be resolved when I
> put
> > in the parallel).
> >
> > Thanks!
> >
> > Perry
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/NOAA-EMC/EVS/pull/94#issuecomment-1553081939>, or
> > unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/APUWOBC7ZC3WYRVJ46RDZ4LXGYR3BANCNFSM6AAAAAAWVW4IXM
> >
> > .
> > You are receiving this because you were mentioned.Message ID:
> > ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/94#issuecomment-1553111740>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFMU3FQWLFKLA74YU53XGYUDLANCNFSM6AAAAAAWVW4IXM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

AliciaBentley-NOAA commented:
Hi Perry,

Unfortunately, I don't see any issues from you yesterday in this thread or
on Github <https://github.com/NOAA-EMC/EVS/pull/94> associated with this
issue! Please let us know what it was. Binbin may not have received it.

Take care,
Alicia
___________________________________
*Alicia M. Bentley, Ph.D.*
Global Verification Project Lead
Model Evaluation Group (MEG) <https://www.emc.ncep.noaa.gov/users/meg/home/>
NOAA/NWS/NCEP/EMC (VPPPG Branch)
Work Phone: ‪(301) 683-3783‬


On Thu, May 18, 2023 at 10:10 AM BinbinZhou-NOAA ***@***.***>
wrote:

> Alicia, Perry,
>
> Sorry I can not catch up what changes need to be made? Please show me
> again.
>
> Thanks!
>
> Binbin
>
>
> On Thu, May 18, 2023 at 10:05 AM Alicia Bentley ***@***.***>
> wrote:
>
> > Hi Perry,
> >
> > If Binbin needs to make changes still, then he should make those changes.
> > Are the changes you are waiting for included in this thread? I think I
> see
> > some requested changes from over a month ago. Would this PR need to be
> > re-tested after those changes are made?
> >
> > Binbin- can you please make the changes that Perry requested of you in
> > early April? Thanks!
> >
> > Take care,
> > Alicia
> > ___________________________________
> > *Alicia M. Bentley, Ph.D.*
> > Global Verification Project Lead
> > Model Evaluation Group (MEG) <
> > https://www.emc.ncep.noaa.gov/users/meg/home/>
> > NOAA/NWS/NCEP/EMC (VPPPG Branch)
> > Work Phone: ‪(301) 683-3783‬
> >
> >
> > On Thu, May 18, 2023 at 9:46 AM PerryShafran-NOAA ***@***.***>
> > wrote:
> >
> > > Hi, Alicia,
> > >
> > > The stats and plots have been validated, but there were a few minor
> > > corrections that I wanted Binbin to make in the PBS function. If that's
> > not
> > > important enough to worry about, I can merge them now. I plan to start
> on
> > > the global_ens PR today. Please let me know if you are OK with merging
> > now
> > > and allowing this issue to be resolved later (it can be resolved when I
> > put
> > > in the parallel).
> > >
> > > Thanks!
> > >
> > > Perry
> > >
> > > —
> > > Reply to this email directly, view it on GitHub
> > > <https://github.com/NOAA-EMC/EVS/pull/94#issuecomment-1553081939>, or
> > > unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/APUWOBC7ZC3WYRVJ46RDZ4LXGYR3BANCNFSM6AAAAAAWVW4IXM
> > >
> > > .
> > > You are receiving this because you were mentioned.Message ID:
> > > ***@***.***>
> > >
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/NOAA-EMC/EVS/pull/94#issuecomment-1553111740>, or
> > unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AQBMPFMU3FQWLFKLA74YU53XGYUDLANCNFSM6AAAAAAWVW4IXM
> >
> > .
> > You are receiving this because you were mentioned.Message ID:
> > ***@***.***>
> >
>
>
> --
>
> Binbin Zhou
>
> Physical Scientist
>
> Lynker at NOAA/NWS/NCEP/EMC
>
> 5830 University Research Ct.
>
> College Park, MD 20740
>
> ***@***.***
>
> 301-683-3683
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/94#issuecomment-1553120420>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APUWOBG25IPMHMFG2FMSA6TXGYUVPANCNFSM6AAAAAAWVW4IXM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

BinbinZhou-NOAA commented:
Perry,

   The  #PBS -N  names for the plotting ecf have been changed and updated
in feature/narre_metplus5

Binbin

On Fri, May 19, 2023 at 8:15 AM PerryShafran-NOAA ***@***.***>
wrote:

> ***@***.**** approved this pull request.
> ------------------------------
>
> In ecf/narre/plots/jevs_narre_past31days_plots.ecf
> <https://github.com/NOAA-EMC/EVS/pull/94#discussion_r1196892638>:
>
> > @@ -0,0 +1,73 @@
> +#!/bin/bash
> +
> +#PBS -N jevs_narre_plot
> +#PBS -j oe
>
> Hi, Binbin,
>
> One last item. Would you change the PBS -N option in your plot job card to
> jevs_narre_past31days_plots and jevs_narre_past90days_plots? Just as
> before, these correspond to the names of the output files, so they need to
> be different so we (and NCO) can differentiate them.
>
> Thanks!
>
> Perry
> ------------------------------
>
> In ecf/narre/plots/jevs_narre_past31days_plots.ecf
> <https://github.com/NOAA-EMC/EVS/pull/94#discussion_r1197881849>:
>
> > @@ -0,0 +1,73 @@
> +#!/bin/bash
> +
> +#PBS -N jevs_narre_plot
> +#PBS -j oe
>
> @BinbinZhou-NOAA <https://github.com/BinbinZhou-NOAA> This is the one
> change that I needed you to address.
>
> Perry
> ------------------------------
>
> In ecf/narre/plots/jevs_narre_past31days_plots.ecf
> <https://github.com/NOAA-EMC/EVS/pull/94#discussion_r1197897608>:
>
> > @@ -0,0 +1,73 @@
> +#!/bin/bash
> +
> +#PBS -N jevs_narre_plot
> +#PBS -j oe
>
> @AliciaBentley-NOAA <https://github.com/AliciaBentley-NOAA> here.
>
> This is a minor issue that doesn't require a re-run. Shelley and I believe
> that we can merge the code now and address this later.
> ------------------------------
>
> In ecf/narre/plots/jevs_narre_past31days_plots.ecf
> <https://github.com/NOAA-EMC/EVS/pull/94#discussion_r1198892765>:
>
> > @@ -0,0 +1,73 @@
> +#!/bin/bash
> +
> +#PBS -N jevs_narre_plot
> +#PBS -j oe
>
> I will approve this and we can deal with this later.
>
> Perry
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/94#pullrequestreview-1431290021>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFIGRBAVZD5RN2EGY43XG5P6BANCNFSM6AAAAAAWVW4IXM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

PerryShafran-NOAA commented:
Thank you.  But your PR has already been approved, so hold on to that and we'll add those in for your next PR.

Thanks!

Perry

AliciaBentley-NOAA commented:
Hi Binbin,

Like Perry said, since the NARRE PR has already been approved and merged,
you can't add new code to it. Please bring the current auth repo into your
global_ens PR and make these #PBS -N names changes in your global_ens PR.

Note: Merge the current auth repo will give you the incorrect version of
run.ver, so please change MET to MET-11.0.2 in your global_ens run.ver to
correct it. Thanks!

Take care,
Alicia
___________________________________
*Alicia M. Bentley, Ph.D.*
Global Verification Project Lead
Model Evaluation Group (MEG) <https://www.emc.ncep.noaa.gov/users/meg/home/>
NOAA/NWS/NCEP/EMC (VPPPG Branch)
Work Phone: ‪(301) 683-3783‬


On Fri, May 19, 2023 at 9:20 AM PerryShafran-NOAA ***@***.***>
wrote:

> Thank you. But your PR has already been approved, so hold on to that and
> we'll add those in for your next PR.
>
> Thanks!
>
> Perry
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/94#issuecomment-1554565319>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APUWOBBIGO5CTOIO5UMLRWDXG5XTNANCNFSM6AAAAAAWVW4IXM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

ShelleyMelchior-NOAA commented:
The auth EVS develop now has the corrected run.ver file.

On Fri, May 19, 2023 at 9:25 AM Alicia Bentley ***@***.***>
wrote:

> Hi Binbin,
>
> Like Perry said, since the NARRE PR has already been approved and merged,
> you can't add new code to it. Please bring the current auth repo into your
> global_ens PR and make these #PBS -N names changes in your global_ens PR.
>
> Note: Merge the current auth repo will give you the incorrect version of
> run.ver, so please change MET to MET-11.0.2 in your global_ens run.ver to
> correct it. Thanks!
>
> Take care,
> Alicia
> ___________________________________
> *Alicia M. Bentley, Ph.D.*
> Global Verification Project Lead
> Model Evaluation Group (MEG) <
> https://www.emc.ncep.noaa.gov/users/meg/home/>
> NOAA/NWS/NCEP/EMC (VPPPG Branch)
> Work Phone: ‪(301) 683-3783‬
>
>
> On Fri, May 19, 2023 at 9:20 AM PerryShafran-NOAA ***@***.***>
> wrote:
>
> > Thank you. But your PR has already been approved, so hold on to that and
> > we'll add those in for your next PR.
> >
> > Thanks!
> >
> > Perry
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/NOAA-EMC/EVS/pull/94#issuecomment-1554565319>, or
> > unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/APUWOBBIGO5CTOIO5UMLRWDXG5XTNANCNFSM6AAAAAAWVW4IXM
> >
> > .
> > You are receiving this because you were mentioned.Message ID:
> > ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/94#issuecomment-1554574768>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMUXYWK4D4CV7QCKP4A5X6DXG5YDXANCNFSM6AAAAAAWVW4IXM>
> .
> You are receiving this because your review was requested.Message ID:
> ***@***.***>
>


-- 

Shelley Melchior

*Lynker*

Contract Branch Lead, Verification/Post-Processing/Product-Generation Branch

NOAA/NWS/NCEP/EMC

301-323-8292 (mobile/SMS)
